src/common/Markdown.js

```diff
@@ -11,8 +12,13 @@ type Props = {
   children: ?string,
 };
 
+const renderMarkdown = markdown => {
+  const domString = md.render(markdown);
+  return domString.replace('<a href=', '<a target="_blank" href=');
```

Comment:
works only for the first link in the text, check the article "Home > Booking > How do I Refer a Friend?"

Reply:
Wow I totally forgot about that.. Thanks!